03-viz.Rmd

```diff
@@ -214,7 +214,7 @@ islands_bar <- ggplot(islands_df, aes(x = landmass, y = size)) +
 islands_bar
 ```
 
-Alright, not bad! This plot is definitely the right kind of visualization, as we can clearly see and compare sizes of landmasses. The major issues are that the smaller landmasses' sizes are hard to distinguish, and the names of the landmasses are obscuring each other as they have been squished into too little space. But remember that the question we asked was only about the largest landmasses; let's make the plot a little bit clearer by keeping only the largest 12 landmasses. We do this using the `top_n` function.
+Alright, not bad! The plot in Figure \@ref(fig:03-data-islands-bar) is definitely the right kind of visualization, as we can clearly see and compare sizes of landmasses. The major issues are that the smaller landmasses' sizes are hard to distinguish, and the names of the landmasses are obscuring each other as they have been squished into too little space. But remember that the question we asked was only about the largest landmasses; let's make the plot a little bit clearer by keeping only the largest 12 landmasses. We do this using the `top_n` function.
```

Comment:
From what I have read about data visualization (see: https://clauswilke.com/dataviz/proportional-ink.html#visualizations-along-linear-axes), we often do not want to start our axes where `ggplot`'s default puts them, as it inflates differences. We should address this here and show them how to change the axes to force them to start from 0.

Reply:
@ttimbers It looks like this barplot does start at 0? Could you clarify this point? thanks!